.pre-commit-config.yaml

```diff
@@ -9,4 +9,5 @@ repos:
     rev: 20.8b1
     hooks:
     -   id: black
+        additional_dependencies: [ 'click==8.0.4' ]
```

Comment:
Would remove this since it isn't needed and could cause confusion by having this library set in two places.

Reply:
Removed